ui/dashboards/src/components/SaveDashboardButton/SaveDashboardButton.tsx

```diff
@@ -67,6 +69,8 @@ export const SaveDashboardButton = ({ onSave, isDisabled, variant = 'contained'
     } else {
       saveDashboard();
     }
+
+    navigate(`/projects/${dashboard.metadata.project}/dashboards/${dashboard.metadata.name}`);
```

Comment:
actually we shouldn't do that directly in the dashboards package since you are taking assumption about how the routing is done.

This should come from the parent, that's why initially it was done in the callback of the method `saveDashboard`